SmartCar/Arduino/sketch_main/sketch_main.ino

```diff
@@ -12,23 +12,27 @@ Isabelle Tornqvist
 Tim Jonasson
 Pontus Laestadius
 Anthony Path
-
 @Version 1.2
 2017-05-11
 Anthony Path: Added battery voltage measurement feature to determine and display battery level.
-
 2017-05-10
 Tim Jonasson: Removed the delay that made it so it doesnt send commands when the value doesnt change and that caused issues since we didnt 
 get the temperature to the app when the temperature didnt change
 Fixed an issue with the flame commands not being sent properly
-
 2017-05-08
 Isabelle Törnqvist: Added code to allow for deactivation of camera servos, X and Y. Refined temperature code. 
 */
 
 const int ONE_WIRE_BUS_PIN = 3; //for temperature sensor
-const int ECHO_PIN = 2;
-const int TRIG_PIN = 4;
+const int ECHO_PIN_FRONT_MID = 2; //front
+const int TRIG_PIN_FRONT_MID = 4;
```

Comment:
Why do all of them use Analog pins but the FRONT_MID one? Are we out of Analog pins? Backwards compatible?